app/javascript/components/Product/Card.tsx

```diff
@@ -20,14 +22,16 @@ export const Card = ({
   footerAction?: React.ReactNode;
   eager?: boolean | undefined;
 }) => (
-  <article className="product-card">
-    <figure>
+  <ProductCard>
+    <ProductCardFigure>
       <Thumbnail url={product.thumbnail_url} nativeType={product.native_type} eager={eager} />
-    </figure>
-    {product.quantity_remaining != null ? <div className="ribbon">{`${product.quantity_remaining} left`}</div> : null}
-    <header>
+    </ProductCardFigure>
+    {product.quantity_remaining !== null ? <Ribbon>{product.quantity_remaining} left</Ribbon> : null}
```

Comment:
```suggestion
    {product.quantity_remaining != null ? <Ribbon>{product.quantity_remaining} left</Ribbon> : null}
```

Specifically when comparing to null I'd rather keep the loose equality - it's helpful in case our type definitions are wrong and it's actually `undefined`